qiskit/primitives/backend_sampler_v2.py

```diff
@@ -15,6 +15,7 @@
 from __future__ import annotations
```

Comment:
Please consider analagous name changes in this file as suggested in `backend_estimator_v2.py`, for instance, switch to the imperative mood preprocessing->preprocess.

Reply:
Yes, I renamed `_postprocessing_pub` with `_postprocess_pub`.